### PR TITLE
Release 1.3.0

### LINKED ISSUE #232: Next release 1.3.0
We have several important changes in the code and on PRs. We should plan to do a release once #215, #219 and #227 are merged

### PR DESCRIPTION
**Pull request by cquiroz**
I think we could do a release, there has been several important changes since 1.2.0

This PR updates the version number and documentation, Let me know if you agree or if you prefer to wait for some other changes

Fixes #232

### CODE REVIEW

**Comment by derekmorr** [comment]
1.3.0 is not ready to release. We still need docs for Formatter. I'm planning on finishing those this weekend.

**Comment by cquiroz** [comment]
Sure, take your time

**Comment by derekmorr** [comment]
I got most of them done on Friday morning, but in the process of writing the example code, I noticed a few nits I should fix. I've been doing pureconfig stuff all day today, so tomorrow I'll have time to get squants stuff done.

Also, once we release 1.3, I'll update pureconfig's squants module to add the new Quantities we added - https://github.com/pureconfig/pureconfig/issues/233

File: README.md [hunk 1]
```diff
@@ -584,7 +584,7 @@ rate2: squants.market.CurrencyExchangeRate = USD/JPY 100.0
 
 scala> // OR
      | val rate3 = JPY(100) -> USD(1)
-rate3: squants.market.CurrencyExchangeRate = USD/JPY 100.0
+rate3: (squants.market.Money, squants.market.Money) = (100.0 JPY,1.0 USD)
```
Comment (cquiroz) [inline README.md]: This change is a bit strange. I think it is related to #217 or #218

Comment (garyKeorkunian) [inline README.md]: I traced this to the 0.13.15 upgrade.  Before that commit, it formats as CurrencyExchangeRate.  After the commit, it formats as a two Money's, which is the underlying type.  That commit only includes minor version upgrades of a few libraries.

Comment (garyKeorkunian) [inline README.md]: And, I shouldn't say "formats".  The actual type seems to be changing as well.  Perhaps some new behavior in Scala?

**Comment by derekmorr** [comment]
I also ran into issues with the scalariform update. It wants to reformat the whole codebase. I had to disable the plugin for my latest PR just to keep the changes small.

**Pull request merged**